flight/Libraries/math/misc_math.c

```diff
@@ -255,6 +255,43 @@ void cubic_deadband_setup(float w, float b, float *m, float *r)
 	*r = *m * powf(w, 3) + b * w;
 }
 
+/**
+ * Perform a linear interpolation over N points
```

Comment:
@kubark42 the mapping is actually constrained to [input_min, input_max] and the number of points is adjustable, so I believe it is a full linear interpolation